src/Record/Record.Model/Records.csproj

```
@@ -12,7 +12,7 @@
 		<EnvironmentSuffix></EnvironmentSuffix>
 		<ReleaseType></ReleaseType>
 		<PackageId>Record</PackageId>
-		<VersionPrefix>9.0.0$(ReleaseType)</VersionPrefix>
+		<VersionPrefix>9.1.0$(ReleaseType)</VersionPrefix>
```

```suggestion
		<VersionPrefix>10.0.0$(ReleaseType)</VersionPrefix>
```
Jeg tror kanskje dette er en breaking change, siden det kommer helt annen rdf ut? Usikker...